src/imitation/scripts/train_preference_comparisons.py

```diff
@@ -134,7 +143,7 @@ def train_preference_comparisons(
 
     reward_net = reward.make_reward_net(venv)
     if agent_path is None:
-        agent = rl_common.make_rl_algo(venv)
+        agent = rl_common.make_rl_algo(venv, rl_kwargs=rl["rl_kwargs"])
```

Comment:
shouldn't it automatically get the `rl_kwargs` as `make_rl_algo` is a `sacred.capture` decorated function?

Reply:
You're probably right, I didn't realize `sacred.capture` behaves that way.